modules/workbench/outputs.tf

```diff
@@ -10,3 +9,7 @@ output "bigquery_views" {
 output "table_names" {
   value = module.reporting.table_names
 }
+
+output "rendered_queries" {
```

Comment:
Please name this to something more specific to indicate *which* rendered queries these are